src/containers/MapControls.jsx

```diff
@@ -68,7 +70,7 @@ class SelectorPanel extends Component {
     //Render!
     return (
       <article className="selector-panel">
-        <section className={(this.props.selectorData.mode !== SelectorData.GUIDED_MODE) ? 'input-subpanel not-selected' : 'input-subpanel' } ref="subPanel_guided">
+        <section className={(this.props.selectorData.mode !== MapSelector.GUIDED_MODE) ? 'input-subpanel not-selected' : 'input-subpanel' } ref="subPanel_guided">
```

Comment:
I'd split ternaries over a few lines to make them easier to read, or even better, define a variable above you can use